pkg/controllers/workgenerator/controller.go

```diff
@@ -438,6 +435,7 @@ func buildAllWorkAppliedCondition(works map[string]*workv1alpha1.Work, binding *
 		Status:             metav1.ConditionFalse,
 		Type:               string(fleetv1beta1.ResourceBindingApplied),
 		Reason:             workNotAppliedReason,
+		Message:            "not all corresponding work objects are applied",
```

Comment:
I'm not sure if message should begin with upper case.

Is there any convention there?

Reply:
I don't think so.